cli/src/commands/router/commands/compose.ts

```diff
@@ -86,6 +86,7 @@ export default (opts: BaseCommandOptions) => {
       argumentConfigurations: result.federationResult.argumentConfigurations,
       federatedSDL: printSchema(result.federationResult.federatedGraphSchema),
       subgraphs: config.subgraphs.map((s, index) => ({
+        id: `${index}`,
```

Comment:
ID can be removed. I think we can match only on the routingUrl. The name can be used for debugging. Ensure the routingUrl is stored normalized without parameters etc...

Reply:
Let's keep ID for the sake of having a unique identifier. I've added url normalization before storing on control plane.

Reply:
Also done